app/web/features/profile/view/userLabels.tsx

```diff
@@ -180,13 +180,20 @@ const AgeAndGenderRenderer = ({ user }: Props) => {
     }
   };
   return (
-    <>
+    <div
+      style={{
+        display: "flex",
+        flexWrap: "wrap",
+        alignItems: "center",
+      }}
+    >
```

Comment:
Please add a class for this to the relevant `useStyles` and put the style there, to align with how we do CSS. You can see examples on other elements on this page, for example if you search `iconStyles`.